### ISSUE TITLE
emulateDevice not working with automationProtocol devtools

### ISSUE THREAD

**Issue by jacekbj** [webdriverio/webdriverio#5050]
**Environment**
 - **WebdriverIO 5.18.6** 
 - **Mode WDIO Testrunner sync**
 - **Node.js 10.17.0**
 - **NPM version 6.11.3**
 - **Chrome 80**
 - **Platform name and version: Debian Buster (Docker)**
 - **Additional wdio packages used: `@wdio/devtools-service, devtools`*

**Config of WebdriverIO**
```
const chromeOptions = {
	args: [
		'--window-size=1600,900',
		'--disable-infobars',
		'--no-sandbox',
		'--disable-notifications',
		'--allow-insecure-localhost',
		'--disable-dev-shm-usage',
		'--disable-extensions',
		'--ignore-certificate-errors',
	],
        // mobileEmulation: { deviceName: 'Galaxy S5' },   - this does not work
};

if (Number(process.env.HEADLESS) === 1) {
        // this flag works
	chromeOptions.headless = true;
}

const chromeCapabilities = {
	browserName: 'chrome',
	'goog:chromeOptions': chromeOptions,
};

exports.config = {
	runner: 'local',
	automationProtocol: 'devtools',
	maxInstances: 3,
	services: ['devtools'],
	reporters: ['spec'],
	framework: 'mocha',
	specs: [`tests/specs/**/*.test.js`],
	capabilities: [chromeCapabilities],
	beforeSuite: () => {
			// this works!
			browser.emulateDevice('Galaxy S5');
		}
	},
};
```

**Describe the bug**
Given:
* `automationProtocol: devtools`,
* headless,
* `mobileEmulation` set in Chrome capabilities;

I expect that:
* emulation is enabled -> screen size is that of Samsung Galaxy S5;

Instead I get:
* the screen size as specified in `--window-size=1600,900`.

**Additional context**
I checked screen size by calling `console.log('browser.execute("window.screen.width")')`.

Interestingly enough, I checked the screen size in "dev" setup with `selenium-standalone` service and it reported `null`.

The workaround with calling `browser.emulateDevice()` in `beforeSuite` works.
1. It is against the docs.
1. `window.screen.width` reporting different values indicate that something might be off with emulation between selenium-standalone and devtools.

**Comment by christian-bromann** [comment]
> * `mobileEmulation` set in Chrome capabilities;

This is a webdriver capability that needs to be translated when using devtools protocol. Sounds like a reasonable enhancement.

**Comment by christian-bromann** [comment]
Fixed in #5280
